test/plain.jl

```diff
@@ -713,16 +713,19 @@ dspace_scal = HDF5.Dataspace(HDF5.h5s_create(HDF5.H5S_SCALAR))
 dspace_norm = dataspace((100, 4))
 dspace_maxd = dataspace((100, 4), max_dims = (256, 4))
 dspace_slab = HDF5.hyperslab(dataspace((100, 4)), 1:20:100, 1:4)
-dspace_irrg = HDF5.Dataspace(HDF5.h5s_combine_select(
-        HDF5.h5s_copy(dspace_slab), HDF5.H5S_SELECT_OR,
-        HDF5.hyperslab(dataspace((100, 4)), 2, 2)))
 
 @test sprint(show, dspace_null) == "HDF5.Dataspace: H5S_NULL"
 @test sprint(show, dspace_scal) == "HDF5.Dataspace: H5S_SCALAR"
 @test sprint(show, dspace_norm) == "HDF5.Dataspace: (100, 4)"
 @test sprint(show, dspace_maxd) == "HDF5.Dataspace: (100, 4) / (256, 4)"
 @test sprint(show, dspace_slab) == "HDF5.Dataspace: (1:20:81, 1:4) / (1:100, 1:4)"
-@test sprint(show, dspace_irrg) == "HDF5.Dataspace: (100, 4) [irregular selection]"
+
+if HDF5.libversion ≥ v"1.12"
+    dspace_irrg = HDF5.Dataspace(HDF5.h5s_combine_select(
```

Comment:
Is it the `HDF5.h5s_combine_select` that's failing here? 

Apparently that should be available on 1.10.7

Reply:
Good point... I was testing with 1.10.4, which is why it fails for me.

Note that 1.10.7 is relatively recent, as it was released in September.